packages/calling/src/CallHistory/types.ts

```diff
@@ -25,6 +30,15 @@ export type UpdateMissedCallsResponse = {
   message: string | null;
 };
 
+export type UCMLinesResponse = {
+  statusCode: number;
+  data: {
+    lines?: UCMLinesApiResponse;
+    error?: string;
```

Comment:
I don't see a test where this error string is being populated. Could we update the error test that returns an error string too?
What would these errors look like?

Reply:
Updated the test case to include the error string as requested.